static/usage/action-sheet/theming/styling/demo.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Action Sheet</title>
+  <link rel="stylesheet" href="../../../common.css" />
+  <script src="../../../common.js"></script>
+  <script type="module" src="https://cdn.jsdelivr.net/npm/@ionic/core@6/dist/ionic/ionic.esm.js"></script>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@ionic/core@6/css/ionic.bundle.css" />
+
+  <style>
+    ion-action-sheet.my-custom-class .action-sheet-group {
+      background: #F58840;
+    }
+
+    ion-action-sheet.my-custom-class .action-sheet-title {
+      color: #EADEDE;
```

Comment:
Can we keep the title/subtitle as white? The gray is a bit hard to read on the orange background.

Also looks like some of the active indicators are not styled: 
![image](https://user-images.githubusercontent.com/2721089/191595688-1afada67-30e9-4354-85f7-2bbdcbc7a83c.png)

Reply:
Also why is the "Delete" button white on iOS and black on MD when the rest of the buttons are always black?

Reply:
Updated based on feedback.

The "Delete" button was white to show how to override the color opinionation we make for iOS mode. I removed this from the example, as the styles do not apply on MD & we do not currently support a CSS variable for developers to easily customize that appearance.